resources/views/bodyMailRegistration.blade.php

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Registro exitoso!</h1>
+    <h2>¿ Che, Hoy donamos ?</h2>
+    <h3>Bienvenido nuevo usuario : {{$data}}, gracias por registrarte.</h3>
+</body>
+</html>
```

Comment:
Missing end of file(EOF)
```suggestion
</html>

```